src/scripts/migrate.ts

```diff
@@ -7,6 +7,7 @@ logger.info('Starting migrations...');
 migrate()
 	.then(() => {
 		logger.info('Migrations complete.');
+		process.exit(0);
```

Comment:
It seems odd that this is needed and makes me curious what was holding it up, but if this works it works.

Reply:
All right this was a good point so I fixed it more correctly!  Gonna ask for a re-review since it's fundamentally different from what you approved.